package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "docs": "scripts/markdown.js",
     "format": "npm-run-all format:*",
-    "format:pkg": "./lib/cli/index.js -wp",
```

Comment:
`-p` was an unknown option causing the positional arguments to be ignored when provided.